R/create-population.R

```diff
@@ -0,0 +1,114 @@
+#' Creates an population using the PKSim Database
+#'
+#' @param populationCharacteristics Characteristics of the population to create as an instance of \code{OriginData}
+#' that are actually distributed parameters
+#'
+#' @return An instance of a population object
+#'
+#' @export
+createPopulation <- function(populationCharacteristics) {
+  validateIsOfType(populationCharacteristics, PopulationCharacteristics)
+
+  populationFactory <- rClr::clrCallStatic("PKSim.R.Api", "GetPopulationFactory")
+  population <- rClr::clrCall(populationFactory, "CreatePopulation", populationCharacteristics$ref)
+  Population$new(population)

```

Comment:
This returns the population object that we use everywhere => we can save to csv, etc.. and it will be loaded into pksim. Full circle